gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/VersionCompletionContributor.java

```diff
@@ -58,17 +61,22 @@ public void addCompletions(
 
                         Project project = parameters.getOriginalFile().getProject();
 
-                        RepositoryLoader.loadRepositories(project).stream()
+                        StreamEx.of(RepositoryLoader.loadRepositories(project))
                                 .flatMap(url -> repositoryExplorer.getVersions(group, dependencyPackage, url).stream())
-                                .map(version -> version.isLatest()
-                                        ? PrioritizedLookupElement.withPriority(
-                                                LookupElementBuilder.create(version)
-                                                        .withTypeText("Latest", true)
-                                                        .withLookupString("latest"),
-                                                Double.MAX_VALUE)
-                                        : LookupElementBuilder.create(version))
+                                .zipWith(IntStream.iterate(0, i -> i + 1).boxed())
+                                .mapKeyValue(this::getLookupElement)
                                 .forEach(resultSet::addElement);
                     }
+
+                    private LookupElement getLookupElement(DependencyVersion version, Integer priority) {
+                        return version.isLatest()
+                                ? PrioritizedLookupElement.withPriority(
+                                        LookupElementBuilder.create(version)
+                                                .withTypeText("Latest", true)
+                                                .withLookupString("latest"),
+                                        Double.MAX_VALUE)
+                                : PrioritizedLookupElement.withPriority(LookupElementBuilder.create(version), priority);
```

Comment:
This is making it so that if the repos are:

* jar
* dist


then all jar completions will appear before dist ones?

If so I'm not sure this is super necessary. It could be from either repo.

Reply:
Ah, it forces the order to be descending too, makes sense.